### ISSUE TITLE
 Environment conditions consideration

### ISSUE THREAD

**Issue by billwuqin** [cpignata/powerbench#8]
2.	Section 3 of draft-cprjgf-bmwg-powerbench said:
“
Enforcing controlled conditions on test conditions (e.g.,
   temperature) may be impractical or unfeasible.  At this stage it is
   unclear whether it is more useful for the benchmark to enforce
   controlled conditions (and if so, which ones) or only specifying how
   those conditions should be measured and reported.

”
This paragraph is not consistent with ETSI-ES-203-136, section 6.1 of ETSI-ES-203-136 said:
“
1) Environmental conditions 
The power measurements shall be performed in a laboratory environment under the following conditions: 
• Room Temperature: 23 °C to 27 °C. 
• Room Relative Humidity: 25 % to 75 %. 
• Atmospheric pressure: 812 hPa to 1 060 hPa. 

”
I believe we should consider these environment conditions such as Room Temperature: 23°Cto 27 °C, Room Relative humidity, Atmospheric Pressure.

**Comment by romain-jacob** [comment]
I have some worries about setting such bounds (even if it is ETSI's position)

- First, the ranges are quite large, and I am not convinced values measured at two ends of the ranges are fairly comparable. This is just a hunch though, but I'd be curious to measure this actually. 
- Many devices I am measuring are in racks, which implies conditions are very different on each side of the device. Where are we supposed to measure? Exhaust? Intake? Both? The ETSI document might specify this, I am not familiar enough with it. Setting a range of environmental conditions is either too much (see next point) or not enough (underspecified).
- But most importantly, reliably measuring those conditions is hard. Temperature is still okay, but humidity and pressure? IMO, having a benchmark that prescribes this is a false-good idea because it makes it really hard for anyone to actually run the benchmark properly. And if no one is using it, the benchmark might just as well not exist.

**Comment by cpignata** [comment]
I also feel it is hard to control humidity and pressure, specifically. We could always have them recorded but not enforced?

**Comment by giuseppefioccola** [comment]
Yes, we can state that the environment conditions (temperature, humidity, atmospheric pressure,...) need to be carefully reported in order to ensure the test replicability. We can simply add these parameters in the section on reporting without imposing specific values.

**Comment by cpignata** [comment]
Added specific environmental conditions (23-27°C, 25-75% RH, 812-1060 hPa) per ETSI ES 203 136 Section 6.1.
